amqp-impl/src/main/java/io/streamnative/pulsar/handlers/amqp/AmqpProtocolHandler.java

```diff
@@ -89,12 +86,7 @@ public String getProtocolDataToAdvertise() {
     @Override
     public void start(BrokerService service) {
         brokerService = service;
-
-        this.amqpTopicManager = new AmqpTopicManager(brokerService.getPulsar());
-        this.exchangeContainer = new ExchangeContainer(this.amqpTopicManager);
-        this.queueContainer = new QueueContainer();
-        this.connectionContainer = new ConnectionContainer(brokerService.getPulsar(), this.exchangeContainer);
-
+        amqpBrokerService = new AmqpBrokerService(service.getPulsar());
```

Comment:
Here, "servcie.getPulsarService()" is better.

Reply:
service.getPulsar() is Pulsar‘s API.